tests/manual-test-cases/Group5-Functional-Tests/5-4-High-Availability.robot

```diff
@@ -17,6 +17,78 @@ Documentation  Test 5-4 - High Availability
 Resource  ../../resources/Util.robot
 Suite Teardown  Nimbus Cleanup  ${list}
 
+*** Keywords ***
+Check VM Info
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.info \*
+    Should Be Equal  ${rc}  0
+    Log To Console  ${output}
```

Comment:
but it just passes ${vm} on wards so you can do Get VM Info    \*
The only problem is if the -r is a problem for you?